.github/workflows/publish.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish Gem
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Publish Gem
+        uses: cadwallion/publish-rubygems-action@master
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          RUBYGEMS_API_KEY: ${{secrets.OSC_RUBYGEMS_API_KEY}}
+          RELEASE_COMMAND: rake release
```

Comment:
You can reference `OSC_ROBOT_RUBYGEMS_TOKEN`.   

We don't seem to have an organization wide github token for the robotic accounts. But I don't think we need it here anyway? We're making tags - it's only pushing to rubygems.

Maybe all we need is:
```
VERSION=$(figure out version)
rake build
gem push "pkg/ood_core-$VERSION.gem"
```

Reply:
I left this comment just as you pushed, but I think it applies anyhow. I think now it may actually be easier to just create the token instead of parsing the GITHUB_REF?

Reply:
Removed need for GITHUB_TOKEN and used `rake build` and `gem push` instead of `rake release`